interface.html

```diff
@@ -35,7 +35,9 @@
   </div>
 
   <div id="screenSection" class="section">
+    {{#unless options.hideAction}}
```

Comment:
I see. But then you might want to do that for all possible actions and not only for `screen`.